Add rng-tools init script

thefallenrat opened the pull request:
The pull request will add an init script for the package `rng-tools`. The init script were based from https://github.com/gentoo/gentoo/tree/master/sys-apps/rng-tools/files/ 

The package `rng-tools` itself is coming from Arch repo, so I don't think it is necessary to import the package to here. Or do you want me import its PKGBUILD too?

Thanks, appreciated it!

physkets commented:
I think the contents of the file 'rngd-confd-4.1' from that gentoo folder will also have to be imported to be placed in the file /etc/conf.d/rngd (or equivalent), because the current file looks very different.

thefallenrat commented:
Ok, I have just imported a `rng-tools` package from archlinux repo, but I modified `confd` to use [gentoo's instead](https://github.com/gentoo/gentoo/blob/master/sys-apps/rng-tools/files/rngd-confd-4.1) and removed systemd service

thefallenrat commented:
> by convention we usually add confd and initd scripts in the openrc package. Could you move the confd part from the PKGBUILD into the galaxy-openrc please?

Done!
